department-of-veterans-affairs/caseflow
# Added convert to virtual for video hearings

Resolves [CASEFLOW-210](https://vajira.max.gov/browse/CASEFLOW-210)

### Description

Gives Hearing Coordinators the ability to convert the hearing request type on any appeal applying the same workflow as was used to convert Travel Board request types implemented in #15314. 

### Acceptance Criteria
- [ ] Users are able to successfully convert hearing request types from Central Office/Video to Virtual

### Testing Plan

1. Shadow user `BVASYELOW`: http://localhost:3000/test/users
2. Navigate to the hearing schedule: http://localhost:3000/hearings/schedule
3. Click the "Schedule Veterans" button
4. Search for "St. Petersburg, FL" in the Regional Office dropdown and navigate to the "AMA Veterans Waiting" tab
5. Select 1 of the Veterans from the list to navigate to their case details page
6. Once the case details page loads, check the actions dropdown for the Convert to Virtual Hearing action and select that
7. Review the conversion page and based on whether you selected the Central Office or another regional office, you should see a message at the top indicating in which queue this request will be
8. Click the Convert to Virtual Hearing button
9. Ensure you receive a success banner and that the action to Convert to Virtual Hearing is no longer available
10. Ensure that the Hearing Request type in the claims header now says Virtual
11. Scroll down and check that there is a ChangeHearingRequestType indicating the conversion
12. Scroll back up and click the Schedule Veteran action and ensure that the Virtual option is preselected

### Code Documentation Updates

### Database Changes
*Only for Schema Changes*
